lib/index.js

```diff
@@ -8,11 +8,15 @@ const servePrivateContentEnabled = (inputs) =>
     return origin && origin.private === true
   })
 
+const unique = (value, index, self) => self.indexOf(value) === index
```

Comment:
Could we use a Set here to ensure unique items? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
👍